install/0000_80_machine-config-operator_01_machineconfig.crd.yaml

```diff
@@ -340,7 +340,7 @@ spec:
                 nullable: true
               kernelType:
                 description: Contains which kernel we want to be running like default
-                  (traditional), realtime
+                  (traditional), realtime, 64k pages
```

Comment:
I am confused, subject says kernel-64K extension but we are making it a kernelType. Looking at rest of the code it seems to be a new kernelType, so please update subject of the PR accordingly.